src/admin/admintask.php

```diff
@@ -14,7 +14,8 @@
 {
 	throw new Exception(JText::_('JERROR_ALERTNOAUTHOR'));
 }
-
+// Require helper file
+JLoader::register('AdmintaskHelper', JPATH_COMPONENT . '/helpers/admintask.php');
```

Comment:
-->Yes ..when we index.php?option=com_categories&extension=com_admintask on page then select  "Tasks" view from sidebar menu ..got the error.